### PR TITLE
Redmine #9113 Create /run/redis for docker users

### PR DESCRIPTION
**Pull request by skwashd**
Without this path redis fails to start under docker.

### CODE REVIEW

File: main/redis/redis.pre-install [hunk 1]
```diff
@@ -2,5 +2,6 @@
 
 addgroup -S redis 2>/dev/null
 adduser -S -D -H -h /var/lib/redis -s /bin/false -G redis -g redis redis 2>/dev/null
+install -o redis -g redis -m 0770 -d /run/redis
```
Comment (andypost) [inline main/redis/redis.pre-install]: when package changes you need to increase pkgrel

Comment (skwashd) [reply main/redis/redis.pre-install]: Good catch. Fixed

Comment (skwashd) [reply main/redis/redis.pre-install]: @clandmeter this is needed for redis to work under docker. Dockerised services should be run in the foreground not as a daemon managed by initd. See [Remind issue 9113](https://bugs.alpinelinux.org/issues/9113) and linked issues for more info.

Comment (andypost) [reply main/redis/redis.pre-install]: See https://git.alpinelinux.org/cgit/aports/commit/?id=c4d4e9aa7abd163db3a9a4c4e6d6ba4f84d3a64f

**Comment by skwashd** [comment]
Yup @jirutka beat me to it. Closing.

**Pull request closed without merging**